### PR TITLE
Extract test utilities to `testutil-gcloud`

### PR DESCRIPTION
**Pull request by dmitrykuzmin**
This PR introduces a new module `testutil-gcloud`.

This module exposes the test utilities previously located in the `gcloud-java` tests, so they can be used in the client projects.

Most notably, the module exposes the `TestDatastoreStorageFactory` which allows to clean up the storage on demand.

Spine version advances to `1.1.3-SNAPSHOT+1`.

### CODE REVIEW

**Comment by dmitrykuzmin** [comment]
@armiol PTAL.

**Comment by armiol** [comment]
@dmitrykuzmin 
```
Please note, that now to build the project locally, you need to copy the spine-dev.json resource into folders $(projectDir)/datastore/src/test/resources and $(projectDir)/testutil-gcloud/src/test/resources.
```
Can we do that in scope of the Gradle build?

File: testutil-gcloud/src/test/java/io/spine/testing/server/storage/datastore/TestDatastoreWrapperTest.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        assertThat(entityReadBeforeClear)
+                .isNotNull();
+
+        // Drop all data.
+        wrapper.dropAllTables();
+
+        // Make sure the entity is no longer present in the Datastore.
+        Entity entityReadAfterClear = wrapper.read(key);
+        assertThat(entityReadAfterClear)
+                .isNull();
```
Comment (armiol) [inline testutil-gcloud/src/test/java/io/spine/testing/server/storage/datastore/TestDatastoreWrapperTest.java]: I think it fits the previous line. It's here and above.

**Review by armiol — COMMENTED** [review]
@dmitrykuzmin LGTM with a comment. Please also see my question in scope of the PR.

**Comment by dmitrykuzmin** [comment]
@armiol PTAL again. 

I've just discovered the credentials copying worked successfully even prior to this PR. No additional actions are needed.

**Review by armiol — APPROVED** [review]

**Pull request merged**